tests/playbooks/centos7_deps.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Install CentOS deps for network system role tests
+  hosts: all
+  vars:
+    - rpmdependencies:
+        - git
+        - python3-pip
+        - python2-pip
+  tasks:
+    - name: Install EPEL
+      package:
+        state: present
+        name: epel-release
+      when:
+        - ansible_distribution == "CentOS"
+        - ansible_distribution_major_version == '7'
```

Comment:
This is required for CentOS 8, too.

Reply:
I will add it. It was working correctly on my CentOS 8 vm without it, though!